test/SourceKit/CodeComplete/complete_from_system.swift

```diff
@@ -0,0 +1,42 @@
+struct MyCollection : Collection {
+  var startIndex: Int { 0 }
+  var endIndex: Int { 0 }
+  func index(after i: Int) -> Int { i + 1 }
+  subscript(position: Int) -> Int { 0 }
+}
+
+func test(col: MyCollection) {
+  col.
+}
+
+// RUN: %sourcekitd-test -req=complete -pos=9:7 %s -- %s -module-name TestMod | %FileCheck %s
+
+// CHECK: {
+// CHECK: key.kind: source.lang.swift.decl.function.method.instance,
+// CHECK: key.name: "makeIterator()",
+// CHECK-NEXT: key.sourcetext: "makeIterator()",
```

Comment:
Do we really care about testing all these fields here? Can we just check is_system specifically?

Reply:
Is there a way to check that the line is in the same item (i.e. before `},` line )?

Reply:
It's not a great solution, but

```
CHECK: key.name: "blah blah blah"
CHECK-NOT: key.name
CHECK: key.is_system: 1
```

Reply:
Cool, that actually works for `is_system: 1` checks. I still haven't found a way to check the item does *not* have `is_system: 1` line.

Reply:
```
// CHECK: key.name: "makeIterator()"
// CHECK-NOT: }
// CHECK: key.is_system: 1
// CHECK: }

// CHECK: key.name: "startIndex"
// CHECK-NOT: }
// CHECK-NOT: key.is_system: 1
// CHECK: },
```
This seems to work.

Reply:
Only danger is that we could add a dictionary in the middle of the response some day, but not sure there is a better solution right now.